.github/workflows/refapp-2x-phoneNumberValidation.yml

```diff
@@ -0,0 +1,41 @@
+name: RefApp 2.x Phone Number validation
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      firefox:
```

Comment:
Is this part of your work, you should you have updated your branch differently, Git reset -hard HEAD~1
Then do follow [this](https://gist.github.com/whoisryosuke/36b3b41e738394170b9a7c230665e6b9)